_layouts/download-3.html

```diff
@@ -330,6 +330,17 @@ <h2>Preview builds</h2>
 		thankYouBackButton.addEventListener('click', () => {
 			thankYouWrapper.style.display = 'none';
 		});
+		thankYouBackButton.addEventListener('keypress', (event) => {
+			// Support button enter and space interactions
+			switch (event.code.toUpperCase()) {
+				case "ENTER":
+				case "SPACE":
+					thankYouWrapper.style.display = 'none';
+					break;
+				default:
+					break;
+			}
+		});
```

Comment:
Why do we need this? Pressing enter already activates the link. Pressing space scrolls the page, but that's normal for web.